src/releaseBusV2/release-bus-v2.operations.test.ts

```
@@ -1501,269 +1360,58 @@ describe('Release Bus v2 exact operation callbacks', () => {
+          repository: 'frontend',
+          environment: 'orchestration',
+          service: null,
+          expected_sha: 'a'.repeat(40),
+          artifact_digest: null,
+          source_ref:
+            'source_ref' in inputs ? String(inputs.source_ref) : undefined,
+          candidate_evidence_mode:
+            'candidate_evidence_mode' in inputs
+              ? (inputs.candidate_evidence_mode as
+                  | 'legacy-whole-train'
+                  | 'strict-aggregate')
```

ESLint

[error] Replace `|·'legacy-whole-train'⏎·················` with `'legacy-whole-train'`

(prettier/prettier)